### PR TITLE
Move configuration options to options doc

### PR DESCRIPTION
**Pull request by jpreese**
For #240

### CODE REVIEW

File: docs/options.md [hunk 1]
```diff
@@ -1,5 +1,21 @@
 # Options
 
+## Configuration
+
+Options for Conftest can be configured on the command line as flag arguments, or as environment variables that are prefixed with `CONFTEST_`.
+
+Conftest also supports a simple configuration file which can be used to store configuration settings for the `conftest` command.
+
+Create a `conftest.toml` configuration file like the following:
```
Comment (garethr) [inline docs/options.md:9]: Worth noting where this is discovered from.

**Comment by jpreese** [comment]
Sorry for the delay on this, team. Addressed @garethr's review feedback.

File: docs/options.md (hunk 1 above)
Comment (jpreese) [reply docs/options.md]: 👍  agreed.

**Review by Blokje5 — APPROVED** [review]
LGTM! The docs are clear and concise!

**Pull request merged**